resource-managers/kubernetes/core/src/main/resources/META-INF/services/org.apache.spark.deploy.rest.kubernetes.DriverServiceManager

```diff
@@ -1,2 +1,18 @@
+#
```

Comment:
Maybe they have an exclusion that ignores license check on that META-INF folder but not this one?  Would you like to match that pattern?